scripts/base_network.py

```diff
@@ -298,9 +293,23 @@ def _set_electrical_parameters_links(links):
     links['p_min_pu'] = -p_max_pu
 
     links_p_nom = pd.read_csv(snakemake.input.links_p_nom)
-    links_p_nom["j"] = _find_closest_links(links, links_p_nom)
-
+    
+    #Filter links that are not in operation anymore    
+    links_p_nom['not_in_operation']=links_p_nom.Remarks.str.contains('Shut down', na=False) | links_p_nom.Remarks.str.contains('Replaced', na=False)
+    
+    links_p_nom = links_p_nom[links_p_nom.not_in_operation == False]
+ 
+    #add column for closest link id
+    links_p_nom["j"] = np.nan
+    
+    #find closest link for all links in links_p_nom        
+    links_p_nom['j'] = _find_closest_links(links, links_p_nom)
+        
+    links_p_nom = links_p_nom.groupby(['j'],as_index=False).agg({'Power (MW)': 'sum',
+                                                                 })    
```

Comment:
```suggestion
```
This seems also obsolete.

Reply:
links_p_nom.groupby is needed, because more than one link belongs to one in the list (example  j 2376)

Reply:
right!